.taskcluster.yml

```diff
@@ -206,6 +210,33 @@ tasks:
           owner: mcastelluccio@mozilla.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - $if: 'tasks_for == "github-pull-request" && "Train on Taskcluster: " in pr_description'
+        then:
+          dependencies:
+            - { $eval: as_slugid("docker_build") }
+          taskId: { $eval: as_slugid("train_on_taskcluster") }
+          created: { $fromNow: "" }
+          deadline: { $fromNow: "1 day" }
+          provisionerId: proj-bugbug
+          workerType: compute-large
+          payload:
+            maxRunTime: 10800
+            image:
+              type: task-image
+              path: public/bugbug/bugbug-base.tar.zst
```

Comment:
Some models require other images (e.g., `bugbug-base-nlp`). We do not need to handle that now, but it would be nice to follow up on this later.

Reply:
I filed an issue https://github.com/mozilla/bugbug/issues/3796